corehq/apps/reports/filters/fixtures.py

```diff
@@ -44,7 +45,9 @@ def filter_context(self):
             'loc_id': loc_id,
             'locations': self.load_locations_json(loc_id),
             'make_optional': self.make_optional,
-            'hierarchy': self.location_hierarchy_config
+            'hierarchy': self.location_hierarchy_config,
+            'path': self.request.path,
+            'have_access_to_features': icds_pre_release_features(user)
```

Comment:
it would be great if this could be more specific in name since this introduces icds dashboard custom code into our main code base. something like `have_access_to_icds_dashboard_features`